tests/test_browser.go

```diff
@@ -355,20 +355,20 @@ func (b *testBrowser) awaitWithTimeout(timeout time.Duration, fn func() error) e
 // returns a pointer to the converted value for convenience.
 //
 // underneath, it uses json.Marshal and json.Unmarshal to do the conversion.
-func convert[T any](t *testing.T, from any, to *T) *T {
-	t.Helper()
+func convert[T any](tb testing.TB, from any, to *T) *T {
```

Comment:
This is done to silent the linter. It's also better because we can use the same in benchmarks if needed.